icu4c/source/common/utext.cpp

```diff
@@ -2394,9 +2394,9 @@ ucstrTextClose(UText *ut) {
 static int64_t U_CALLCONV
 ucstrTextLength(UText *ut) {
     if (ut->a < 0) {
-        // null terminated, we don't yet know the length.  Scan for it.
+        // null terminated, we don't yet know the length. Scan for it.
         //    Access is not convenient for doing this
-        //    because the current interation postion can't be changed.
+        //    because the current interaction postion can't be changed.
```

Comment:
Should be 'iteration' rather than 'interaction'

Reply:
My bad 😅 Thanks!